src/common/vars.ts

```diff
@@ -29,6 +30,54 @@ export const defaultTheme = "lens-dark" as string;
 export const defaultFontSize = 12;
 export const defaultTerminalFontFamily = "RobotoMono";
 export const defaultEditorFontFamily = "RobotoMono";
+export const normalizedPlatform = (() => {
+  switch (process.platform) {
+    case "darwin":
+      return "darwin";
+    case "linux":
+      return "linux";
+    case "win32":
+      return "windows";
+    default:
+      throw new Error(`platform=${process.platform} is unsupported`);
+  }
+})();
+export const normalizedArch = (() => {
+  switch (process.arch) {
+    case "arm64":
+      return "arm64";
+    case "x64":
+    case "amd64":
+      return "x64";
+    case "386":
+    case "x32":
+    case "ia32":
+      return "ia32";
+    default:
+      throw new Error(`arch=${process.arch} is unsupported`);
+  }
+})();
+
+export function getBinaryName(name: string, { forPlatform = normalizedPlatform } = {}): string {
+  if (forPlatform === "windows") {
+    return `${name}.exe`;
+  }
+
+  return name;
+}
+
+export const resourcesDir = onceCell(() => (
```

Comment:
Why do we need `onceCell`?

Reply:
Because `process` is not defined when the `vars` are imported by the `webpack.*.ts` files and therefore throw. This allows for the code of instantiation to happen at first use not at definition time.

Reply:
Global state problem? @jansav

Reply:
Yes global state, but mainly we should not be importing application code in our build files like this.

I think a PR to remove the imports of `common/vars` from our webpack files would be a good thing.